lib/honeybadger.js

```diff
@@ -74,6 +76,11 @@ function configure(opts) {
     if (configOpts.indexOf(opt) < 0) { throw(new Error('Invalid config option: ' + opt)); }
     this[opt] = opts[opt];
   }
+  // Defualt value for reportUncaught is true.
+  // If reportUncaught is anything but false, treat it as defualt.
+  if (this.reportUncaught === false && uncaughtExceptionHandler) {
```

Comment:
Why do we need this? The previous handler shouldn't be installed if `reportUncaught` is false? :slightly_smiling_face:

Reply:
:+1: 

Can we set the default value of reportUncaught around: https://github.com/GlideMe/honeybadger-node/blob/680864286fbd1d6442150ddacc7a98a3b831863c/lib/honeybadger.js#L37

Reply:
i'll explicitly add the true value there. should i also make it settable with an environment variable so that the singleton wont start up with the listener? i can even remove it from the config and only make it reliant on an environment variable, which might be cleaner.

Reply:
Sounds good :+1: Please add the default value
For the sake of consistency we should avoid going the environment variable route :slightly_smiling_face: .